core/cmd/terra_chains_commands_test.go

```diff
@@ -0,0 +1,132 @@
+package cmd_test
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli"
+	null "gopkg.in/guregu/null.v4"
+
+	"github.com/smartcontractkit/chainlink-terra/pkg/terra/db"
+
+	"github.com/smartcontractkit/chainlink/core/cmd"
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+)
+
+const terraChainID = "Chainlinktest-99"
```

Comment:
I would put it in the fixture, or autogenerate a random one in each test. Otherwise you will get lock contention.